libc/docs/math/index.rst

```diff
@@ -136,7 +136,7 @@ Basic Operations
 +------------------+------------------+-----------------+------------------------+----------------------+------------------------+------------------------+----------------------------+
 | fabs             | |check|          | |check|         | |check|                | |check|              | |check|                | 7.12.7.3               | F.10.4.3                   |
 +------------------+------------------+-----------------+------------------------+----------------------+------------------------+------------------------+----------------------------+
-| fadd             | N/A              | |check|         | |check|                | N/A                  | |check|                | 7.12.14.1              | F.10.11                    |
+| fadd             | N/A              | |check|                                 | N/A                  |                        | 7.12.14.1              | F.10.11                     |
```

Comment:
Kind of nitpicky, but technically I think this should be in its own PR, since:

> Once you have a patch ready, it is time to submit it. The patch should:
> - [...]
> - not contain any unrelated changes
> - be an isolated change. Independent changes should be submitted as separate patches as this makes reviewing easier.

https://llvm.org/docs/Contributing.html#how-to-submit-a-patch

I don't know if this gets a pass for being such a small change or not. @lntue

Reply:
will fix this

Reply:
I think it's ok for this small docs update as it's tangent to this PR, just like sometime we fix some thing right next to what we update.

Reply:
Right.